pkgs/applications/science/electronics/sv-lang/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchFromGitHub
+, boost182
+, catch2_3
+, cmake
+, fmt
+, python3
+}:
+
+let
+  # dependency for this package has been removed in master (i.e. next release)
+  unordered_dense = stdenv.mkDerivation rec {
+    version = "2.0.1";
+    pname = "unordered_dense";
+    src = fetchFromGitHub {
+      owner = "martinus";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "sha256-9zlWYAY4lOQsL9+MYukqavBi5k96FvglRgznLIwwRyw=";
+    };
+    nativeBuildInputs = [
+      cmake
+    ];
+  };
+
+in
+stdenv.mkDerivation rec {
+  # its another name `slang` coincides with https://www.jedsoft.org/slang/
```

Comment:
```suggestion
```

Reply:
May I ask where is the change? Two lines look identical.

Reply:
The change is the removal of the comment. pname's can clash, that's not a problem